neoload/README.md

```diff
@@ -0,0 +1,43 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the out-of-the-box Dashboard or [Metrics Explorer][7].
```

Comment:
Added additional information on what this integration tracks based off the demo today. Please edit if I misunderstood

```suggestion
[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.

With the NeoLoad integration, you can track Neoload test performance metrics to:

- Correlate application performance with load testing metrics in Neoload.
- Analyze and visualize NeoLoad metrics in Datadog like throughput, errors, and performance using the out-of-the-box Dashboard or [Metrics Explorer][7].
```